CHANGELOG.md

```diff
@@ -20,6 +20,15 @@
 
 - HashMaps in UDL now support a default value with an empty map ([#2539](https://github.com/mozilla/uniffi-rs/pull/2539)).
 
+XXX - thiss change log is missing 29.2??
```

Comment:
I think I forgot to open a PR after releasing 0.29.2.  Here it is now though.: https://github.com/mozilla/uniffi-rs/pull/2546/